modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/layout/layout-screen/LayoutScreen.tsx

```diff
@@ -22,6 +23,12 @@ import './LayoutScreen.scss';
 const LayoutScreen: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 	return (
 		<div className="layout-tab">
+			<ClayAlert displayType="info" title="Info:">
```

Comment:
Hey @eli-rlima, could you use language key instead of this title?

Reply:
Thanks, @interaminense! I forgot this.